components/event-bus/cmd/subscription-controller/cmd/manager/main.go

```diff
@@ -56,7 +61,15 @@ func main() {
 		log.Error(err, "unable to add event activation APIs to scheme")
 		os.Exit(1)
 	}
+	if err := messagingV1Alpha1.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "unable to add event activation APIs to scheme")
+		os.Exit(1)
+	}
 
+	if err := evapisv1alpha1.AddToScheme(mgr.GetScheme()); err != nil {
```

Comment:
Please remove the space before this block, and add one after it.

Reply:
Done.